ui/src/app/views/account-creation/component.spec.ts

```diff
@@ -0,0 +1,194 @@
+import {DebugElement} from '@angular/core';
+import {ComponentFixture, fakeAsync, TestBed, tick} from '@angular/core/testing';
+import {FormsModule} from '@angular/forms';
+import {By} from '@angular/platform-browser';
+import {UrlSegment} from '@angular/router';
+import {RouterTestingModule} from '@angular/router/testing';
+
+import {ClarityModule} from '@clr/angular';
+
+import {ProfileService} from 'generated';
+
+import {ProfileServiceStub} from 'testing/stubs/profile-service-stub';
+import {ServerConfigServiceStub} from 'testing/stubs/server-config-service-stub';
+
+import {
+  simulateEvent,
+  simulateInput,
+  updateAndTick
+} from '../../../testing/test-helpers';
+import {ServerConfigService} from '../../services/server-config.service';
+
+import {AccountCreationSuccessComponent} from '../account-creation-success/component';
+import {AccountCreationComponent} from '../account-creation/component';
+import {InvitationKeyComponent} from '../invitation-key/component';
+import {LoginComponent} from '../login/component';
+import {PageTemplateSignedOutComponent} from '../page-template-signed-out/component';
+import {RoutingSpinnerComponent} from '../routing-spinner/component';
+
+
+describe('AccountCreationComponent', () => {
+  let profileServiceStub: ProfileServiceStub;
+  let fixture: ComponentFixture<AccountCreationComponent>;
+  beforeEach(fakeAsync(() => {
+    profileServiceStub = new ProfileServiceStub();
+    TestBed.configureTestingModule({
+      imports: [
+        RouterTestingModule,
+        FormsModule,
+        ClarityModule.forRoot()
+      ],
+      declarations: [
+        LoginComponent,
+        AccountCreationComponent,
+        AccountCreationSuccessComponent,
+        InvitationKeyComponent,
+        PageTemplateSignedOutComponent,
+        RoutingSpinnerComponent
+      ],
+      providers: [
+        { provide: LoginComponent, useValue: {}},
+        { provide: InvitationKeyComponent, useValue: {}},
+        { provide: ProfileService, useValue: profileServiceStub },
+        {
+          provide: ServerConfigService,
+          useValue: new ServerConfigServiceStub({
+            gsuiteDomain: 'fake-research-aou.org'
+          })
+        }]
+    }).compileComponents().then(() => {
+      fixture = TestBed.createComponent(AccountCreationComponent);
+      tick();
+    });
+    tick();
+  }));
+
+  it('handles selecting password', fakeAsync(() => {
+    updateAndTick(fixture);
+    expect(fixture.componentInstance.passwordOffFocus).toBeUndefined();
```

Comment:
Probably makes more sense to initialize these properties to `true` in the component, rather than leaving them uninitialized. This test would make more sense then.